pkg/compiler/debug.go

```diff
@@ -585,11 +585,8 @@ func (di *DebugInfo) ConvertToManifest(o *Options) (*manifest.Manifest, error) {
 	for _, method := range di.Methods {
 		if method.IsExported && method.IsFunction && method.Name.Namespace == di.MainPkg {
 			mMethod := method.ToManifestMethod()
-			for i := range o.SafeMethods {
-				if mMethod.Name == o.SafeMethods[i] {
-					mMethod.Safe = true
-					break
-				}
+			if slices.Contains(o.SafeMethods, mMethod.Name) {
```

Comment:
Can be `mMethod.Safe = slices.Contains(o.SafeMethods, mMethod.Name)` without an `if`